docs/src/main/sphinx/optimizer/pushdown.rst

```diff
@@ -271,3 +271,92 @@ FETCH FIRST N ROWS``.
 
 Implementation and support is connector-specific since different data sources
 support different SQL syntax and processing.
+
+Find two similar Top-N queries in the following sections.
```

Comment:
Maybe @mosabua can suggest some better opening statement?

Reply:
For example, you can examine at two queries that implement a Top-N behavior in the following section.

Reply:
WDYT? `For example, you can find two queries to learn how to identify Top-N pushdown behavior in the following section.`